README.md

```diff
@@ -1,56 +1,78 @@
-<a name="readme-top"></a>
-# Microsoft Teams Integration / Dataminer Bot
-
-This guideline is about how to integrate Dataminer into Microsoft Teams with automation scripts.
+<a id="readme-top"></a>
+
+# Chat Integration
+
+This guideline is about how to integrate Dataminer into business communication platforms with automation scripts.
+
+<details>
+<summary>Table of Contents</summary>
+  <ul>
+    <li><a href="#overview">Overview</a></li>
+    <li><a href="#prerequisites">Prerequisites</a></li>
+    <li><a href="#installation">Installation</a></li>
+    <li><a href="#examples">Examples</a></li>
+    <li><a href="#help">Help</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#version-history">Version History</a></li>
+    <li><a href="#license">License</a></li>
+  </ul>
+</details>
 
 ## Overview
 
-There are two main parts in this guideline:
- * Adding the automation scripts & memory files to your Dataminer system.
- * Granting admin consent in the DCP admin app.
+### Teams
+
+There are two main parts to the Chat Integration:
+
+- Adding the automation scripts & memory files to your Dataminer System.
+- Granting admin consent in the DCP admin app.
```

Comment:
Move this line to Prerequisites > Teams and remove the rest from overview
Put the Table of contents under Overview subtitle